x-pack/plugins/observability_ai_assistant/public/components/prompt_editor/prompt_editor.tsx

```diff
@@ -142,6 +142,8 @@ export function PromptEditor({
   useEffect(() => {
     if (hidden) {
       onChangeHeight(0);
+    } else if (containerRef.current) {
+      onChangeHeight(containerRef.current.clientHeight);
```

Comment:
Because I memoized the callback being passed in, this wasn't being called as often due to less renders so had to make sure it's called at least when this flag switches.

_The ResizeablePanel causes more renders to happen which caused the focus bug due to missing memoization_